index.bs

```diff
@@ -297,8 +317,28 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
       then return true.</li>
       <li>Otherwise, for each <var>item</var> in the <a>allowlist</a>:
         <ol>
-          <li>If <var>item</var> is [=same origin-domain=] with
-          <var>origin</var>, then return true.</li>
+          <li>If <var>item</var> is a <a>serialized-origin</a> that's [=same origin-domain=] 
+          with <var>origin</var>, then return true.</li>
+          <li>Else if <var>item</var> is a <a>serialized-origin-with-wildcard-subdomain</a>
+          then:</li>
+          <ol>
+            <li>If the <var>tuple origin</var> lacks a <a>host</a> with a <a>registrable domain</a>
+            return false.<li>
+            <li>Set <var>originWithoutWildcard</var> to be equal to <var>item</var> with the sequence
+            of <a>code points</a> "*." removed at the start of the <a>host</a>.</li>
+            <li>If <var>originWithoutWildcard</var> is [=same origin-domain=] </li> with
+            <var>origin</var>, then return false.</li>
+            <li>Set <var>originCandidate</var> to be equal to <var>origin</var> with the
+            left most domain label (and the dot after it) removed.</li>
```

Comment:
I think you'd need stricter definition of what"left most domain label removed" is. (that is, loop over the code points until you find a dot, and substring from that point, or something similar)